llvm/utils/TableGen/RegisterInfoEmitter.cpp

```diff
@@ -962,7 +962,7 @@ void RegisterInfoEmitter::runMCDesc(raw_ostream &OS, CodeGenTarget &Target,
 
   OS << "extern const MCRegisterDesc " << TargetName
      << "RegDesc[] = { // Descriptors\n";
-  OS << "  { " << RegStrings.get("") << ", 0, 0, 0, 0, 0 },\n";
+  OS << "  { " << RegStrings.get("") << ", 0, 0, 0, 0, 0, 0 },\n";
```

Comment:
I'm not entirely sure what the purpose of this line is. Since it's printed before register descriptors are emitted, my assumption is that it's displaying the general layout of the MCRegisterDesc struct. As I am changing MCRegisterDesc to hold 7 instead of 6 elements, this change seemed relevant in order to keep the structure consistent.

Reply:
It would be the first entry in the array, which is presumably used to make entry '0' a special, invalid register ID. It worked before your update because if you omit entries in a struct initializer, they get zero-initialized.

Reply:
I understand why it worked without this change, my question was more regarding whether or not this is even worth changing. Like you said, the bool IsConstant gets zero-initialised anyway. I only added this in to keep things looking consistent, which I wasn't sure was enough of a reason to commit this change.

Reply:
Updating it for consistency seems a good enough reason to me. 👍

Reply:
Alright, in it stays.